modules/graphdb/dns.tf

```diff
@@ -1,4 +1,6 @@
 resource "aws_route53_zone" "graphdb_zone" {
+  count = var.graphdb_node_count > 1 ? 1 : 0
```

Comment:
Probably lots of things would break, and I'm not sure if `graphdb_node_count = 0` is a legit case, but in some places the checks are `graphdb_node_count != 1`.